src/boolean_result.cpp

```diff
@@ -14,9 +14,12 @@
 
 #include <algorithm>
 #include <array>
-#include <iostream>
 #include <map>
 
+#ifdef MANIFOLD_DEBUG
+#include <iostream>
```

Comment:
Good catch! How long has that been there? Should we have this conditional include of string stuff in `utils.h` and then have a rule against including these headers anywhere else?

Reply:
Sounds good.